Aztec/Classes/TextKit/TextView.swift

```diff
@@ -309,17 +309,21 @@ open class TextView: UITextView {
 
     // MARK: - Overwritten Properties
     
-    /// Overwrites Typing Attributes:
-    /// This is the (only) valid hook we've found, in order to (selectively) remove the [Blockquote, List, Pre] attributes.
-    /// For details, see: https://github.com/wordpress-mobile/AztecEditor-iOS/issues/414
+    /// The reason why we need to use this, instead of `super.typingAttributes`, is that iOS seems to sometimes
+    /// modify `super.typingAttributes` without us having any mechanism to intercept those changes.
     ///
-    override open var typingAttributes: [String: Any] {
+    private var myTypingAttributes = [String: Any]()
+    
+    override open var typingAttributes: [String : Any] {
         get {
-            ensureRemovalOfParagraphAttributesAfterSelectionChange()
-            return super.typingAttributes
+            return myTypingAttributes
         }
+        
         set {
+            // We're still setting the typing the paren't typing attributes in case it's used directly
+            // by any iOS feature.
```

Comment:
Small detail: Seems to be something wrong with the redaction of the comment. Could you double check it? :)

Reply:
Addressed.